Exercitii/index.js

```diff
@@ -0,0 +1,26 @@
+
+console.log("EXERCITIUL 1")
+
+//let a
+
+console.log(" 'a' a fost declarat")
+
+console.log('valoarea lui a este ${a}')
+
+//let a ="10"
+
+console.log("lui a i s-a atribuit 10")
+
+console.log('valoarea lui a este ${a}')
+
+//const a = "20"
+
+console.log("a si-a schimbat valoarea")
+
+console.log('valoarea lui a este ${a}')
+
+let a = "a + 1"
```

Comment:
Din nou, aici nu trebuie reatribuita variabila, ci doar trebuie sa assignam valoarea care exista inainte in variabila + 1, si anume
```js
a = a + 1; //acum a va avea valoarea precedenta + 1
```